### PR TITLE
created wrapper for empty space

### PR DESCRIPTION
**Pull request by sksanjoo2**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/<TODO>

## Screenshot / UI changes

Describe the UI changes / make a screenshot

## Testing instructions

Please describe with a few words how to reproduce the problem / test the feature

### CODE REVIEW

**Comment by martinfrances107** [comment]
@sksanjoo2 

Hello welcome to the project.

I have a couple of comments 

a) When filling out the new pull request template, if you add the issue number to the end of the URL then it will bind the PR to the issue.  .. like this.

> Issue
> https://github.com/jsdrupal/drupal-admin-ui/issues/508
> Screenshot / UI changes

b) When I look at the code regarding the use InlineMessage.

here is a line to blog 
https://codeburst.io/using-promises-and-error-boundaries-for-data-loading-within-your-react-redux-app-219f8d3a6d26

shows that the type of error as passed into componentDidCatch can be anything. In this case a promise

```
  componentDidCatch(error, info) {
    // if we have a promise then we can deal with it
    if(error instanceof Promise) {
      // we have a promise so lets update our state
      this.setState({ isLoading: true });
      // once the promise has resolved we can update our state again and grab the data
      error.then((data) => this.setState({ isLoading: false, data }));
    } 
```

looking at our implementation of ErrorBoundary passes the error into InlineMessage message property 


```
  render() {
    if (this.state.error) {
      const pe = new PrettyError();

      return (
        <InlineMessage
          message={pe.render(this.state.error)}
          messageSeverity={MESSAGE_SEVERITY_ERROR}
        />
      );
    }
```

So in short, we do not know in advance the type of message.

Can I ask you to add a conditional statement so that we only wrap strings with a P element.

something like 

```
if (typeof error === 'string' )
  <p>{message}</p>
else
  {message}
```

Anyway I hope this logic makes sense - I hope this helps.
